tests/model_registry/utils.py

```diff
@@ -446,6 +484,31 @@ def get_mr_service_objects(
     return services
 
 
+def get_mr_configmap_objects(
+    base_name: str,
+    namespace: str,
+    client: DynamicClient,
+    teardown_resources: bool,
+    num: int = 1,
+    db_backend: str = "mariadb",
```

Comment:
what is the difference here?

Reply:
Thanks for catching, this should be removed.